### PR TITLE
Change bend and taper, and fix stl ordering.

### PR DESCRIPTION
**Pull request by pentacular**
Bend now takes a radius for the reference circle to fit the curve to.
Taper takes four factors of z.
Stl should have triangles leading with the minimal vertex.

### CODE REVIEW

File: api/shape/define.js [hunk 1]
```diff
@@ -14,73 +14,69 @@ export const defThreejsMaterial = (name, definition) =>
...
 
-export const defGrblDynamicLaser = (
-  name,
-  {
-    cutDepth = 0.2,
-    diameter = 0.09,
-    jumpPower = 0,
-    power = 1000,
-    speed = 1000,
+export const GrblDynamicLaser = ({
```
Comment (BarbourSmith) [inline api/shape/define.js:36]: Are you thinking about doing some laser stuff?

File: nb/documentation/interactions_with_geometry/interactions_with_geometry.md [hunk 2]
```diff
@@ -164,43 +164,47 @@ Remesh can be used to break up the segments of a shape allowing it to be distort
...
 
 ![Image](interactions_with_geometry.md.14.png)
 
+A rectangle bent into a ring with a central radius of 25
+
+---
+### Rotate
+A shape can be rotated about the origin using the .rx(), .ry(), and .rz() commands.
+These take the number of turns as an argument .rz(1/8) would rotate the shape by 1/8th of a rotation.
+Multiple turns can be provided, which will produce one result per turn.
```
Comment (BarbourSmith) [inline nb/documentation/interactions_with_geometry/interactions_with_geometry.md:185]: That's neat, I wasn't aware of that feature

**Review by BarbourSmith — APPROVED** [review]
As always this looks great to me 😀

File: nb/documentation/interactions_with_geometry/interactions_with_geometry.md (hunk 2 above)
Comment (pentacular) [reply nb/documentation/interactions_with_geometry/interactions_with_geometry.md:185]: Yeah, the fly in the ointment is that there's no way to post-process on a per turn basis, but I think a parallel op might work.

e.g., s.rz(0/1, 1/2).op(color('blue'), color('red')) might work, except it needs something to distinguish between an op on the whole shape and an op on s.n(0), s.n(1), s.n(2).

File: api/shape/define.js (hunk 1 above)
Comment (pentacular) [reply api/shape/define.js:36]: Laser stuff was working ok, but this is just clean up.

I'll get the laser set up in a while and make sure it's working with the simplified system.

**Pull request merged**